nixos/modules/services/web-servers/caddy/default.nix

```diff
@@ -376,7 +376,7 @@ in
         ReadWriteDirectories = cfg.dataDir;
         StateDirectory = mkIf (cfg.dataDir == "/var/lib/caddy") [ "caddy" ];
         LogsDirectory = mkIf (cfg.logDir == "/var/log/caddy") [ "caddy" ];
-        Restart = "on-abnormal";
+        Restart = "on-failure";
```

Comment:
Should `Restart` be set in Nixpkgs?  The upstream [intentionally](https://github.com/caddyserver/dist/pull/77#issuecomment-947837243) do not add it.

Reply:
A quick `grep` of the web-servers directory shows that every other one does this.